src/lib/CustomRulePathManager.ts

```diff
@@ -148,24 +168,29 @@ export class CustomRulePathManager {
     for (const p of paths) {
       let stats;
       try {
+        this.logger.trace(`Fetching stats for path ${p}`);
         stats = await this.fileHandler.stats(p);
       } catch (e) {
         throw SfdxError.create('@salesforce/sfdx-scanner', 'add', 'errors.invalidFilePath', [p]);
       }
       if (stats.isFile()) {
         if (p.endsWith(".jar")) {
           // Simple filename check for .jar is enough.
+          this.logger.trace(`Adding JAR directly provided as a path: ${p}`);
           classpathEntries.push(p);
         }
       } else if (stats.isDirectory()) {
         // Always add directories, which may contain classes
+        this.logger.trace(`Adding directory directly provided as a path: ${p}`);
```

Comment:
My changes will cause a conflict with this. Trivial to resolve, though.

Reply:
Let me remove this line as well to see if we can avoid conflict.